### PR TITLE
add social-auth-core

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: wheels/social-auth-core/meta.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+
+type: wheel
+name: social-auth-core
+version: 3.0.0+gx0
```
Comment (nsoranzo) [inline wheels/social-auth-core/meta.yml]: Is this for Galaxy? 19.01 requires 1.5.0, while `dev` requires 3.1.0 .

Comment (bgruening) [reply wheels/social-auth-core/meta.yml]: Yes for Galaxy. It seems the 3.1.0 was never released on github: https://github.com/python-social-auth/social-core/releases

I will use 3.1.0+gx0 then.

File: wheels/social-auth-core/meta.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+
+type: wheel
+name: social-auth-core
+version: 3.1.0+gx0
+src:
+    - https://github.com/python-social-auth/social-core/archive/d3d9c81bdc8b873ff5610e853dc7e9c8f1133e35.zip
+
```
Comment (nsoranzo) [inline wheels/social-auth-core/meta.yml]: Build is failing, maybe you need something like https://github.com/galaxyproject/starforge/blob/master/wheels/build/wheels.yml#L179

**Comment by nsoranzo** [comment]
Green, can you double-check @bgruening ?

**Comment by bgruening** [comment]
Looks good to me, thanks for the help.
But I'm not sure this has actually done anything. I only see `No wheel changes for builder 'universal', terminating`.

**Comment by nsoranzo** [comment]
You need to look to at the "Pure Python 2/3 wheels" jobs.

I've set `purepy: true` instead of `universal: true` because I think `social-auth-core` cannot build universal wheels, see https://pypi.org/project/social-auth-core/#files where there are separate py2 and py3 pure Python wheels.

**Comment by bgruening** [comment]
I see, ok this looks good.

What is the procedure in this repo. Waiting for @natefoo or can we proceed?

**Pull request merged**